training_record/services.py

```diff
@@ -141,6 +143,76 @@ def create_campus_records_from_excel(file):
 
         return len(records)
 
+    @staticmethod
+    def department_admin_review(record_id, is_approved, user):
+        '''Department admin review the off-campus training record.'''
```

Comment:
Add comments to describe function signature like `Parameters`, `Return` for this service method.